admin.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+// must be run within Dokuwiki
+if(!defined('DOKU_INC')) die();
+
+class admin_plugin_translation extends DokuWiki_Admin_Plugin {
+    function forAdminOnly() {
+        return false;
+    }
+
+    function handle() {
+    }
+
+    function html() {
+
+        /** @var helper_plugin_translation $helper */
+        $helper = plugin_load('helper', "translation");
+        $default_language = $helper->defaultlang;
+
+        /** @var Doku_Renderer_xhtml $xhtml_renderer */
+        $xhtml_renderer = p_get_renderer('xhtml');
+
+        echo "<h1>" . $this->getLang("menu") . "</h1>";
+        echo "<table id='outdated_translations'>";
```

Comment:
Can you give the table the class ```inline```, please? That way it get's the templates default styling